src/Pages/Playlists/Components/VideoCart.js

```diff
@@ -0,0 +1,35 @@
+import { usePlaylist } from "../../../Context";
+import { removeVideoFromPlaylist } from "../../../ApiService";
+export const VideoCart = ({ _id, author, thumnailMedium, playlistId }) => {
+  const { playlistDispatch } = usePlaylist();
+
+  const removeHandler = async (id, playlistId) => {
+    const encodeToken = localStorage.getItem("encodedToken");
+    const res = await removeVideoFromPlaylist(playlistId, id, encodeToken);
+    console.log(res);
+    playlistDispatch({
```

Comment:
remove console

Reply:
thanks point out this . i will remove this next commit

Reply:
done